inception-project/inception
# #4674 - Ability to select annotators and documents when calculating agreement

**What's in the PR**
- Added ability to select annotators and documents when calculating agreement
- Changed layout of agreement page

**How to test manually**
* Use the updated agreement page
* Run without any documents / users selected (should calculate on all then)
* Run with limiting to certain documents / users

**Automatic testing**

**Documentation**
